apps/communications/dialer/test/marionette/lib/views/activities/views.js

```diff
@@ -19,6 +19,18 @@ ActivitiesView.prototype = {
       var contacts = this.client.loader.getAppClass('contacts');
       contacts.switchToCreateNewContactActivity();
       return contacts;
+  },
+
+  pickAddToContact: function(){
```

Comment:
This function is very close to `pickCreateNew()`. Let's factorize the behavior in a function like:
```js
_openElement: function(accessorToTap) {}
```